### DISCUSSION TITLE
Cê tá bem, Netflix?

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#439] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/ce-ta-bem-netflix/

**Comment by one-eyed-willy** [comment]
Nem falo nada sobre a "inteligência" do catálogo da Netflix :weary:

**Comment by maryon-telvanni** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/ce-ta-bem-netflix/#comment_1160657028>):

> Nem falo nada sobre a "inteligência" do catálogo da Netflix 😩

Esses dias "Olha quem está falando" de 1989 apareceu entre os filmes "Em Alta" do catálogo. Mesmo sendo bem antigo, é um filme bacana que eu já assisti, várias vezes, há muito tempo. Não quero assistir de novo, mas marquei como "Gostei" pra ver se alguma outra coisa antiga que talvez eu não conheça pudesse aparecer. Foi clicar no "Gostei" que na hora o tal filme ficou em destaque na minha página de Filmes. :rage:
